tests/types/test_object_types.py

```diff
@@ -121,3 +124,30 @@ class WishfulThinking:
     field: StrawberryField = WishfulThinking._type_definition.fields[0]
 
     assert field.type is EU
+
+
+def test_raises_error_when_using_type_with_a_not_class_object():
```

Comment:
This file (and directory of tests) is used for testing the `StrawberryType` logic of fields. We need to revisit the organization of tests in the future, but for now I would create a new test file in `tests/objects` for these